package/raphael.js

```diff
@@ -1996,29 +1996,119 @@ R._path2string = function () {
     return this.join(",").replace(p2s, "$1");
 };
 
-function repush(array, item) {
-    for (var i = 0, ii = array.length; i < ii; i++) {
-        if (array[i] === item) {
-            return array.push(array.splice(i, 1)[0]);
-        }
-    }
-}
-
 var cacher = R._cacher = function (f, scope, postprocessor) {
+    var blank = null,
+        start = null,
+        end = null,
+        cache = {},
+        count = 0;
+
     function cachedfunction() {
-        var _args = (0, _raphael.getArrayCopy)(arguments),
-            arg = arraySlice.call(_args, 0),
+        var arg = (0, _raphael.getArrayCopy)(arguments),
             args = arg.join('\u2400'),
-            cache = cachedfunction.cache = cachedfunction.cache || {},
-            count = cachedfunction.count = cachedfunction.count || [];
+            newEndStr,
+            newEnd,
+            cur,
+            prev,
+            next,
+            nextStr,
+            newNode;
+
+        /**
+         * Special case. for blank string, store separately.
+         */
+        if (args === '') {
```

Comment:
@priyanjitdey94 Assign a keyword for ' ' and try the generalised flow. It might work.